src/platform/Beken/SoftwareUpdateManagerImpl.h

```diff
@@ -0,0 +1,89 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2019 Google LLC.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#if CHIP_DEVICE_CONFIG_ENABLE_SOFTWARE_UPDATE_MANAGER
+
+#include <platform/internal/GenericSoftwareUpdateManagerImpl.h>
+#include <platform/internal/GenericSoftwareUpdateManagerImpl_BDX.h>
+
+namespace chip {
+namespace DeviceLayer {
+
+/**
+ * Concrete implementation of the SoftwareUpdateManager singleton object for the
+ * Beken platform.
+ */
+class SoftwareUpdateManagerImpl final : public SoftwareUpdateManager,
```

Comment:
This is obsolete interface, not used anymore.

Reply:
Thanks, we will remove it.